Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1224,6 +1230,54 @@ def get_github_actions_usage():
     return_outputs(readable_output=human_readable, outputs=ec, raw_response=usage_result)
 
 
+def get_file_content_from_repo():
+    """Gets the content of a file from GitHub.
+    """
+    params = demisto.params()
+    args = demisto.args()
+
+    owner_name = params.get('user')
+    repository_name = params.get('repository')
+    file_path = args.get('file_path')
+    branch_name = args.get('branch_name')
+    media_type = args.get('media_type', 'raw')
+    create_war_room_entry = argToBoolean(args.get('create_war_room_entry', True))
+
+    url_suffix = f'/repos/{owner_name}/{repository_name}/contents/{file_path}'
+    if branch_name:
+        url_suffix += f'?ref={branch_name}'
+
+    headers = {
+        'Authorization': "Bearer " + TOKEN,
+        'Accept': f'application/vnd.github.VERSION.{media_type}',
+    }
+
+    file_data = http_request(method="GET", url_suffix=url_suffix, headers=headers, is_raw_response=True)
+
+    file_processed_data = {
+        'Path': file_path,
+        'Content': file_data,
+        'MediaType': media_type,
+    }
+    if branch_name:
+        file_processed_data['Branch'] = branch_name
+
+    if create_war_room_entry:
+        readable_output = tableToMarkdown(f'File {file_path} successfully fetched.', file_processed_data, removeNull=True)
+    else:
+        readable_output = f'File {file_path} successfully fetched.'
```

Comment:
either way, you are creating a War-Room entry, what does it matter?
I expected an option to return the file as a file entry